common/scripts/tracing.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Are you planning on placing the tracing.sh in common-files as well for all repos and this is just a test, or is this going to be an override for the istio repo?

Reply:
It will go in common, this is just for testing